wmagent_status.py

```diff
@@ -0,0 +1,106 @@
+import glob
+import numpy as np
+import matplotlib.pyplot as plt
+from pprint import pprint 
+import sys
+import json
+from datetime import datetime, timedelta
+import time
+import matplotlib.cm as cm
+
+from collections import defaultdict 
+from operator import itemgetter
+import matplotlib.ticker as ticker
+
+import os
+
+os.system("xrdcp -r root://eoscms.cern.ch//eos/cms/store/logs/prod/recent/agent_crashes .")
```

Comment:
after merging this, I will change the path accordingly, the unified nodes have eos mount, so that we can just glog.glob it